### PR TITLE
fix internal info page

### PR DESCRIPTION
**Pull request by j-konopka**
change category and license
uniform changelog

@plentymarkets/team-order-payment 

https://plentymarkets.kanbanize.com/ctrl_board/50/cards/146947/details/

### CODE REVIEW

File: src/Methods/PayUponPickupPaymentMethod.php [hunk 1]
```diff
@@ -72,14 +73,14 @@ public function getName(string $lang = 'de'): string
...
-    * Get PayUponPickup Fee
+    * Return Payment Method Fee
     *
     * @return float
     */
     public function getFee(): float
     {
 
-        return 0.00;
+        return -10.00;
```
Comment (raphael-adam) [inline src/Methods/PayUponPickupPaymentMethod.php]: Why this fixed value of -10? And why is there code after the return?

Comment (j-konopka) [reply src/Methods/PayUponPickupPaymentMethod.php]: Oh this was just a mistake. I used this value to test an other case 😄 
Fixed now.

Comment (raphael-adam) [reply src/Methods/PayUponPickupPaymentMethod.php]: Ok, but still strange that there is code after the return.

**Review by raphael-adam — APPROVED** [review]

File: src/Methods/PayUponPickupPaymentMethod.php (hunk 1 above)
Comment (j-konopka) [reply src/Methods/PayUponPickupPaymentMethod.php]: Year thats right. I think we still left the code there as an example.

**Review by estafilarakis — APPROVED** [review]

**Review by pwieczorek-plenty — APPROVED** [review]
Approved by Testing

**Pull request merged**